doc/ref/plotting_options/styling.ipynb

```diff
@@ -0,0 +1,742 @@
+    "          '#6baed6','#4292c6','#2171b5','#084594']\n",
+    "\n",
+    "depth_colors = {\n",
+    "    'Shallow': '#FFC300',\n",
+    "    'Intermediate': '#C70039',\n",
+    "    'Deep': '#581845'\n",
+    "}\n",
+    "\n",
+    "mpl_cmap = colormaps[\"OrRd\"]\n",
+    "\n",
+    "plot1 = df.hvplot.scatter(\n",
+    "    x=\"lon\",\n",
+    "    y=\"lat\",\n",
+    "    c=\"mag\",\n",
+    "    cmap=\"plasma_r\",\n",
+    "    frame_width=width,\n",
+    "    title=\"Earthquakes using named cmap\"\n",
+    ")\n",
+    "\n",
+    "plot2 = df.hvplot.scatter(\n",
+    "    x=\"lon\",\n",
+    "    y=\"lat\",\n",
+    "    c=\"mag\",\n",
+    "    cmap=Blues8_r,\n",
+    "    frame_width=width,\n",
+    "    title=\"Earthquakes using custom list cmap\"\n",
+    ")\n",
+    "\n",
+    "plot3 = df.hvplot.scatter(\n",
+    "    x=\"lon\",\n",
+    "    y=\"lat\",\n",
+    "    c=\"mag\",\n",
+    "    cmap=mpl_cmap,\n",
+    "    frame_width=width,\n",
+    "    title=\"Earthquakes using MPL colormap object\"\n",
+    ")\n",
+    "\n",
+    "plot4 = df.hvplot.scatter(\n",
+    "    x=\"lon\",\n",
+    "    y=\"lat\",\n",
+    "    c=\"depth_class\",\n",
+    "    cmap=depth_colors,\n",
+    "    frame_width=width,\n",
+    "    title=\"Earthquake depths using dict keys\"\n",
+    ")\n",
+    "\n",
+    "(plot1 + plot2 + plot3 + plot4).cols(2)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "861a158b-be6a-46ad-a556-a8cc8d9c069d",
+   "metadata": {},
+   "source": [
+    "::: {tip}\n",
+    "**Prefer `Colorcet` for color mapping**\n",
+    "\n",
+    "While you can use any valid Matplotlib or Bokeh colormap with `cmap`, we recommend using the [`Colorcet`](https://colorcet.holoviz.org) library when possible. It provides a wide selection of perceptually accurate, well-tested colormaps optimized for data visualization.\n",
+    "\n",
+    "`Colorcet` is also natively integrated with other HoloViz tools like [HoloViews](https://holoviews.org) and [Datashader](https://datashader.org), which means:\n",
+    "- No extra setup is required\n",
+    "- Colormaps work consistently across all visualizations\n",
+    "- Defaults like `glasbey_category10` and `kbc_r` come from `Colorcet`\n",
+    "\n",
+    "This makes it a reliable and visually accessible choice for both scientific and presentation-quality plots.\n",
+    ":::"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1c34a309-d7c0-423e-adc7-9743f3d7a331",
+   "metadata": {},
+   "source": [
+    "(option-colormap)=\n",
+    "## `colormap`\n",
+    "\n",
+    "Alias for [`cmap`](option-cmap) above."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3290b419-10a3-4d2b-ac99-0e50c644fbcc",
+   "metadata": {},
+   "source": [
+    "(option-color_key)=\n",
+    "## `color_key`\n",
+    "\n",
+    "The `color_key` option defines a categorical color mapping, primarily used with `datashade=True`. It maps distinct categories in your dataset to specific colors.\n",
```

Comment:
Can we say `color_key` should only be used exclusively with `datashade=True`? Until we figure out why it needs to be used instead of `cmap` (still not clear to me, maybe the answer is in your colormapping notebook?).